src/test/java/de/tum/in/www1/artemis/ArchitectureTest.java

```diff
@@ -216,4 +222,32 @@ public void check(JavaMethod item, ConditionEvents events) {
             }
         };
     }
+
+    private static final Set<String> SQL_KEYWORDS = Set.of("SELECT", "UPDATE", "SET", "DELETE", "DISTINCT", "EXISTS", "FROM", "WHERE", "LEFT", "OUTER", "INNER", "JOIN", "FETCH",
+            "TREAT", "AND", "OR", "AS", "ON", "ORDER", "BY", "ASC", "DSC", "GROUP", "COUNT", "SUM", "AVG", "MAX", "MIN", "IS", "NOT", "FALSE", "TRUE", "NULL", "LIKE", "IN",
+            "BETWEEN", "HAVING", "EMPTY", "MEMBER", "OF", "UPPER", "LOWER", "TRIM");
```

Comment:
Could you reference your source for that so we can update this or check if there are new keywords quickly?